### PR TITLE
#81103 Add validation to form

### PR DESCRIPTION
**Pull request by LubomirMaron**

### CODE REVIEW

File: server/ERNI.PBA.Server.Business/Commands/Requests/AddRequestCommand.cs [hunk 1]
```diff
@@ -33,8 +33,14 @@ public class AddRequestCommand : Command<PostRequestModel>, IAddRequestCommand
 
         protected override async Task Execute(PostRequestModel parameter, ClaimsPrincipal principal, CancellationToken cancellationToken)
         {
-            var currentYear = DateTime.Now.Year;
+            
+            if (parameter.Amount < 1)
```
Comment (menyet) [inline server/ERNI.PBA.Server.Business/Commands/Requests/AddRequestCommand.cs:37]: what about 0.5??

File: client/src/app/requests/requestAdd/requestAdd.component.ts [hunk 2]
```diff
@@ -18,42 +18,36 @@ import { BudgetTypeEnum } from '../../model/budgetTypeEnum';
...
+        let budgetId = this.budgetId;
+        let title = this.title;
+        let amount = this.amount;
+        let year = this.date.year;
 
-        this.busyIndicatorService.start();
+        let date = new Date(this.date.year, this.date.month, this.date.day);
 
         let requestData = { budgetId, title, amount, date } as NewRequest;
+        console.log(requestData);
```
Comment (menyet) [inline client/src/app/requests/requestAdd/requestAdd.component.ts:50]: do you need this log?

File: server/ERNI.PBA.Server.Business/Commands/Requests/AddRequestCommand.cs (hunk 1 above)
Comment (jakubsturc) [reply server/ERNI.PBA.Server.Business/Commands/Requests/AddRequestCommand.cs:37]: Do we have a clear requirement here? If not, I think the 1Euro rule is ok. Nevertheless, error message is misleading.

Comment (LubomirMaron) [reply server/ERNI.PBA.Server.Business/Commands/Requests/AddRequestCommand.cs:37]: > 
> 
> Do we have a clear requirement here? If not, I think the 1Euro rule is ok. Nevertheless, error message is misleading.

Why misleading ? Its good to be positive :) :)

Comment (LubomirMaron) [reply server/ERNI.PBA.Server.Business/Commands/Requests/AddRequestCommand.cs:37]: > 
> 
> what about 0.5??

Change to greater than 0

**Pull request merged**